[Authentication][Signoutflowadded][1570772] (#2918)
MicrosoftDocs/msteams-docs: msteams-platform/tabs/how-to/authentication/auth-silent-aad.md

### Authenticate

> If ADAL has an unexpired token cached for the user, use the token. Alternately, attempt to get a token silently by calling `acquireToken(resource, callback)`. ADAL.js calls your callback function with the requested token, or gives an error if authentication fails.

Reviewer: ```suggestion
If ADAL has a token cached for the user that has not expired, use that token. Alternately, get a token silently by calling `acquireToken(resource, callback)`. ADAL.js calls the callback function with the requested token, or gives an error if authentication fails.
```

    }
}
```

### Handle sign out flow

> Use the following code to handle sign out flow in AAD Auth:

Reviewer 2: Please give some insight on sign out flow, "While we do logout for teams tab or Bot, we clear the current session"

```javascript

> function logoutV1() {

Reviewer 2: Please rename this Method name to "logout" remove V1

localStorage.clear();

> window.location.href = "@Url.Action("ChooseAuth", "Home")";

Reviewer 2: Please replace chooseAuth with `<<Action Name>>` and Home to `<<Controller Name>>`
